wildfly82/README.md

```diff
@@ -0,0 +1,24 @@
+Artificer: Wildfly 8.2
+======================
+
+Artificer is an metadata and software artifact repository comprised of a common data model, multiple interfaces, powerful tools, and exensibility. It implements the [OASIS S-RAMP specification](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=s-ramp) which defines the data model, query language, and an Atom REST binding.
+
+In addition to implementing the spec, Artificer provides a suite of powerful, flexible, and extensible capabilities, described below.
+
+Artificer is 100% open source -- contributions are welcome!
+
+Visit [artificer.jboss.org](http://artificer.jboss.org) to learn more.
+
+## Usage
+
+To start Artificer
+
+    docker run -it artificer/wildfly82[:tag]
```

Comment:
Hi, this should be:
Does it have a tight dependency on wildfly 8.2? Does it run on wildfly 8.1? Would it probably run on wildfly 8.3 if there is one? If answers is not, I would recommend to name it wildfly, as the rest of the projects. If yes, then it is fine.
But really, as wildfly comes with the Dockerfile, it should be enough to name it wildfly.

What it is not fine is the final name of the image, that will have jboss prefix :-D

jboss/artificer-wildfly82[:tag]

or jboss/artificer-wildfly[:tag] 

And as without port mapping this does not make sense, you should include either -p (as below) or -P.